ui/components/app/terms-of-use-popup/terms-of-use-popup.js

```diff
@@ -39,6 +40,12 @@ export default function TermsOfUsePopup({ onAccept }) {
     });
   };
 
+  const handleScroll = (e) => {
+    setShouldShowScrollButton(
+      e.target.scrollHeight - e.target.scrollTop !== e.target.clientHeight,
```

Comment:
Really cool, great work!

Reply:
Thank you for the contribution @dominikrudzki 

Would it be possible to add a debounce to the scroll event?  This could be very expensive from a performance perspective with the number of times `onScroll` is called.  

Thank you!